optimade/server/entry_collections/entry_collections.py

```diff
@@ -338,6 +338,13 @@ def handle_query_params(
         if getattr(params, "page_offset", False):
             cursor_kwargs["skip"] = params.page_offset
 
+        # page_number
+        if getattr(params, "page_number", False):
+            if isinstance(params.page_number, int):
+                cursor_kwargs["skip"] = (params.page_number - 1) * cursor_kwargs[
+                    "limit"
+                ]
+
```

Comment:
Should probably error if page_number and page_offset are both supplied, or use only the offset? (i.e. just replace this with `elif`)

Reply:
I have added an error message when both page_number and page_offset are set.